client/boot/index.js

```diff
@@ -343,7 +343,7 @@ function reduxStoreReady( reduxStore ) {
 	}
 
 	if ( config.isEnabled( 'desktop' ) ) {
-		require( 'lib/desktop' ).init();
+		// require( 'lib/desktop' ).init();
```

Comment:
@scruffian and I are getting `electron is not defined` errors when we load this module (which has `electron` as a dependency).